analyses/fusion_filtering/results/rec_genes_participant_histology_level.tsv

```diff
@@ -0,0 +1,396 @@
+FusionName	broad_histology	count
```

Comment:
From a quick look, this seems like it might be a duplicate of `analyses/fusion_filtering/results/rec_fusions_participant_histology_level.tsv`